js/initcontext.go

```diff
@@ -200,9 +200,9 @@ func (i *InitContext) requireModule(name string) (goja.Value, error) {
 	if !ok {
 		return nil, fmt.Errorf("unknown module: %s", name)
 	}
-	if modV2, ok := mod.(modules.IsModuleV2); ok {
-		instance := modV2.NewModuleInstance(&moduleInstanceCoreImpl{ctxPtr: i.ctxPtr})
-		return i.runtime.ToValue(toESModuleExports(instance.GetExports())), nil
+	if modV2, ok := mod.(modules.Module); ok {
```

Comment:
`modV2` now sounds ... bad, but I either need to shadow `mod` or not name one of them `mod`/`module` :(

Reply:
Just `m`? :smile: `modV2` would be confusing...